src/overlay/template.jsx

```diff
@@ -95,7 +96,7 @@ export function Overlay({
       return;
     }
 
-    if (isIos()) {
+    if (isIos() || isIpadOs()) {
```

Comment:
I realize that this will also affect Chrome on iPad which might not be the experience that we want. Will look into covering this scenario.

Reply:
Tested the Chrome on iPad scenario and it was working as expected.